README.md

```diff
@@ -35,20 +35,41 @@ This requires a free FMOD account.
 
 ### Windows
 
-- Download the "FMOD Engine" package for Windows.
+- Download the "FMOD Studio" and ""FMOD Engine" package for Windows.
 - Install the package.
 - You need the following 4 files in the root of your rust project:
     - `api/core/lib/x64/fmod.dll`
     - `api/core/lib/x64/fmod_vc.lib`
     - `api/studio/lib/x64/fmodstudio.dll`
     - `api/studio/lib/x64/fmodstudio_vc.lib`
 
+### MacOS
+
+- Download the "FMOD Studio" and "FMOD Engine" package for MacOS.
+- Move these files:
+    - `api/core/lib/libfmod.dylib`
+    - `api/studio/lib/libfmodL.dylib` (debugging only)
+    - `api/studio/lib/libfmodstudio.dylib`
+    - `api/studio/lib/libfmodstudioL.dylib` (debugging only)  
```

Comment:
I don't think we've mentioned the debugging (or Logging) version of the libs before, should we explain how they are used? (Could be a follow-up PR)

Reply:
I have seen that libfmod automatically loads one or the other depending on some cargo emv var. I will further investigate in a follow-up and remove the mention here for now.